flink-streaming-java/src/test/java/org/apache/flink/streaming/api/graph/StreamingJobGraphGeneratorTest.java

```diff
@@ -1693,6 +1723,10 @@ public JobGraph createGraphWithMultipleInputs(boolean chain, String... inputName
         transform.setChainingStrategy(
                 chain ? ChainingStrategy.HEAD_WITH_SOURCES : ChainingStrategy.NEVER);
 
+        DataStream<Long> dataStream = new DataStream<>(env, transform);
+        // do not chain with sink operator.
+        dataStream.rebalance().addSink(new DiscardingSink<>()).name(sinkName);
```

Comment:
We can use the `MultipleConnectedStreams` here.
```
  new MultipleConnectedStreams(env)
          .transform(transform)
          .rebalance()
          .addSink(new DiscardingSink<>())
          .name(sinkName);
```

Reply:
The way it is written here should be fine and achieves the same effect as mentioned in the comment. So I ignored this comment.

Reply:
The way it is written here should be fine and achieves the same effect as mentioned in the comment. So I ignored this comment.